src/IstanbulTestReporter.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var DefaultTestReporter = require('./DefaultTestReporter');
+var istanbul = require('istanbul');
+var collector = new istanbul.Collector();
+var reporter = new istanbul.Reporter();
+
+function IstanbulTestReporter(customProcess) {
+  this.process = customProcess || process;
+}
+
+IstanbulTestReporter.prototype = new DefaultTestReporter();
+
+IstanbulTestReporter.prototype.onTestResult =
+function(config, testResult, aggregatedResults) {
+  DefaultTestReporter.prototype.onTestResult.call(
+    this, config, testResult, aggregatedResults
+  );
+
+  if (config.collectCoverage) {
+    collector.add(testResult.coverage);
```

Comment:
This appears to crash if the tests failed. This can be fixed by changing the if statement to:
if (config.collectCoverage && testResult.coverage) {